pkg/domain/infra/abi/play.go

```diff
@@ -114,6 +117,19 @@ func (ic *ContainerEngine) PlayKube(ctx context.Context, body io.Reader, options
 	report := &entities.PlayKubeReport{}
 	validKinds := 0
 
+	// when no network options are specified, create a common network for all the pods
+	if len(options.Networks) == 0 {
+		_, err := ic.NetworkCreate(
+			ctx, nettypes.Network{
+				Name:       kubeDefaultNetwork,
+				DNSEnabled: true,
+			},
+		)
+		if err != nil && err != nettypes.ErrNetworkExists {
```

Comment:
```suggestion
		if err != nil && !errors.Is(err,nettypes.ErrNetworkExists) {
```
due the possible go error wrapping, errors should be checked with errors.Is()